20181010_Software_Dev_practices_in_R/README.md

```diff
@@ -0,0 +1,454 @@
+---
+title: Software Development Tools for Analysts using R
+author: Will Bowditch
+width: 1600
+height: 1200
+---
+# Why use software development tools? 🤔
+
+- As analysts we care about getting the right answer, so the code needs to be correct and maintainable
+- Writing code can lead to lots of avoidable mistakes. There are a whole bunch of tools and practises from software development that can help us avoid mistakes
```

Comment:
Writing code can lead to lots of avoidable mistakes. There are a whole bunch of tools and practises from software development that can help to prevent them.